assets/src/stories-editor/index.js

```diff
@@ -201,6 +202,7 @@ subscribe( async () => {
 		maybeUpdateAutoAdvanceAfterMedia( block );
 		maybeSetTagName( block );
 		maybeSetInitialSize( block );
+		maybeUpdateFontSize( getBlock( block ) );
```

Comment:
I'm wondering if there's a way to use it only after inserting the template instead and only for the newly added template blocks? Otherwise, it will run for all the blocks with every change, every time each block element will be changed in size to check for the font, etc. Thoughts? 

This isn't related to the incorrect font size though.

Reply:
The `insertBlock` action (called by the template inserter) is async, so you cannot directly call something right afterwards.

We could, however, update our own store after `insertBlock` is called to set an internal state variable ourselves. In the subscribe handler we would then only do something if that variable is set.

some pseudo code off the top of my head:

```
insertBlock(...)
dispatch('amp/stories').setHasTemplateBeenInserted( pageClientId )
...

if ( select('amp/stories').hasTemplateBeenInserted() ) {
pageClientId.innerBlocks.foreach(updateFontSize)
dispatch('amp/stories').clearHasTemplateBeenInserted( pageClientId )
}

Reply:
I was thinking that one option could be using `await` for `insertBlocks`.

Better to use a state though.

Reply:
I don‘t think that works, IIRC it‘s not a Promise. It just dispatches an action and returns immediately. Redux in the background then handles the action

Reply:
I actually tried meanwhile and it does seem to work, however, not sure that's the best approach anyway.

Basically adding this instead of using `insertBlock` directly. 
```
async insertTemplate( template ) {
		const { insertBlock, selectBlock } = this.props;
		await insertBlock( template );
	}
```


However, I'm thinking that maybe we could instead add the font update to `componentDidMount` of the Text block:
```
componentDidMount() {
		maybeUpdateFontSize( this.props );
	}
```
Tested very briefly that this seems to fix the font size issue as well. _Testing more thoroughly_

Reply:
Nope, it does not resolve that issue, it displayed correctly with just one of the templates, no clue why.